smac/epm/gaussian_process.py

```diff
@@ -262,15 +264,30 @@ def _predict(self, X_test: np.ndarray, full_cov: bool = False) -> typing.Tuple[n
             raise Exception('Model has to be trained first!')
 
         X_test = self._impute_inactive(X_test)
-        mu, var = self.gp.predict(X_test, return_cov=True)
-        var = np.diag(var)
 
-        # Clip negative variances and set them to the smallest
-        # positive float value
-        var = np.clip(var, VERY_SMALL_NUMBER, np.inf)
+        predict_kwargs = {'return_cov': False, 'return_std': True}
```

Comment:
Could this be moved into the else block? It looks like it belongs rather there than here.